config-connector/solutions/sql/kpt/mysql-public/sqldatabase.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: sql.cnrm.cloud.google.com/v1beta1
+kind: SQLDatabase
+metadata:
+  name: mysql-public-database
+spec:
+  charset: UTF8
+  collation: en_US.UTF8
```

Comment:
I tried to test it but found the error saying `Unknown collation: 'en_US.UTF8'`. Check the [public doc](https://cloud.google.com/sql/docs/mysql/create-manage-databases#create) and found the default values. So could you change to the following spec:
```
charset: utf8
collation: utf8_general_ci
```